README.rst

```diff
@@ -20,41 +29,70 @@ simtools
 .. image:: https://codecov.io/gh/gammasim/simtools/branch/master/graph/badge.svg?token=AYAIRPARCH
    :target: https://codecov.io/gh/gammasim/simtools
 
+Simulation tools and applications for the `Cherenkov Telescope Array (CTAO) <www.cta-observatory.org>`_.
```

Comment:
This link is not working in the documentation page, I think because we are missing the 'https://'.

Reply:
Fixed.